.github/scripts/test-info.mjs

```diff
@@ -832,6 +838,7 @@ const components = {
  * @property {string?} nodeJsVersion If set, installs the specified Node.js version
  * @property {string?} mongoDbVersion If set, installs the specified MongoDB version
  * @property {string|string[]?} sourcePkg If set, sets the specified source package
+ * @property {boolean} requireRavenDBCredentials If true, sets RavenDB license environment variable
  */
 
 /**
```

Comment:
The matrix is generated minus the step to set the environment variable it'll need to be added.

Reply:
@nmalocic Hi! This is the comment I was referring to